demo/admin/src/dashboard/LatestBuilds.tsx

```diff
@@ -1,19 +1,12 @@
-import { gql } from "@apollo/client";
-import { useTableQuery } from "@comet/admin";
+import { gql, useQuery } from "@apollo/client";
 import { LatestBuildsDashboardWidget } from "@comet/cms-admin";
 import * as React from "react";
 
 import { GQLLatestBuildsQuery, GQLLatestBuildsQueryVariables } from "./LatestBuilds.generated";
 
 export const LatestBuilds: React.FC = () => {
-    const tableQuery = useTableQuery<GQLLatestBuildsQuery, GQLLatestBuildsQueryVariables>()(LATEST_BUILDS, {
-        resolveTableData: (data) => ({
-            data: data.builds,
-            totalCount: 5,
-        }),
-    });
-
-    return <LatestBuildsDashboardWidget tableQuery={tableQuery} />;
+    const { data, loading, error } = useQuery<GQLLatestBuildsQuery, GQLLatestBuildsQueryVariables>(LATEST_BUILDS);
```

Comment:
Looks to me like the builds query is defined in the library: 
https://github.com/vivid-planet/comet/blob/24754e4067c8cdf10d891e9343026a75509d4114/packages/api/cms-api/src/builds/builds.resolver.ts#L46

So, you could move this query into the LatestBuildsDashboardWidget instead of defining it in the Demo project

Reply:
Good point 👍🏼